### ISSUE TITLE
znx is not utilizing the version of lsblk included in the AppImage

### ISSUE THREAD

**Issue by UriHerrera** [Nitrux/znx#29]
znx outputs this message after trying to initialize the device `lsblk: columna desconocida: PATH,PARTLABEL`. This problem occurs because the version of lsblk needed for znx to work is 2.33.1; however the version that comes by default in Nitrux is 2.31.1 which does not have support for the column `PATH`.

```
uri@AX370-Gaming-5:~/Downloads$ sudo ./znx init /dev/sdb
znx: Wiping /dev/sdb. Please wait.
/dev/sdb: se han borrado 8 bytes en el desplazamiento 0x00000200 (gpt): 45 46 49 20 50 41 52 54
/dev/sdb: se han borrado 8 bytes en el desplazamiento 0x39adffe00 (gpt): 45 46 49 20 50 41 52 54
/dev/sdb: se han borrado 2 bytes en el desplazamiento 0x000001fe (PMBR): 55 aa
Creating new GPT entries.
GPT data structures destroyed! You may now partition the disk using fdisk or
other utilities.
Setting name!
partNum is 0
Setting name!
partNum is 1
The operation has completed successfully.
lsblk: columna desconocida: PATH,PARTLABEL
lsblk: columna desconocida: PATH,PARTLABEL
````

The development AppImage of znx, however, does include version 2.33.1 of lsblk but znx is not making use of this binary, instead, it is using the binary from the host system.

```
uri@AX370-Gaming-5:~/Downloads/squashfs-root/usr/bin$ strings lsblk | grep "2\.3"
util-linux 2.33.1
GLIBC_2.3
GLIBC_2.3.4
SMARTCOLS_2.30
SMARTCOLS_2.33
```

Note: _Manually installing `utils-linux` 2.33.1 in the host system (Nitrux) does fix the problem_.

**Comment by luis-lavaire** [comment]
[This comment](https://github.com/AppImage/AppImageKit/issues/933#issuecomment-488947501) could be the most simple solution to this issue. :smiley:

**Comment by luis-lavaire** [comment]
Solved in 44d6523.

**Comment by luis-lavaire** [comment]
This issue may be as well related to [this](https://github.com/Nitrux/znx-gui/issues/7) issue.
